admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -151,11 +151,17 @@ object DatabaseTablesDAO {
           field <- fields
           fieldName = field.name
           data = {
-            if (settings.photoColumn.contains(fieldName)) {
-              // TODO: we're doing this operation two times (get id string), find another way
-              val rowId = resultSet.getString(settings.primaryKeyField)
-              s"$baseUrl/admin/images/$tableName/$rowId"
-            } else resultSet.getString(fieldName)
+            val maybe = settings.columnTypeOverrides.find(_._1 == fieldName)
+            maybe
+              .map { case (columnName, customDataType) =>
+                customDataType match {
+                  case CustomDataType.BinaryImage =>
+                    val rowId = resultSet.getString(settings.primaryKeyField)
+                    s"$baseUrl/admin/images/$tableName/$columnName/$rowId"
+                  case CustomDataType.Binary | CustomDataType.Text => resultSet.getString(columnName)
```

Comment:
same, also, I'd believe that we can return an empty item for now (add a ToDo to fix this)